core/controllers/learner_dashboard.py

```diff
@@ -189,6 +200,14 @@ class LearnerDashboardFeedbackThreadHandler(base.BaseHandler):
     """Gets all the messages in a thread."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'thread_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
```

Comment:
Are there any rules for thread IDs that we can apply here?

Reply:
I have looked into the codebase but found nothing extra check for thread_id.
Please suggest me something, thank you.

Reply:
https://github.com/oppia/oppia/blob/develop/core/storage/feedback/gae_models.py#L55

Reply:
Thank you.